ossfuzz/compress_fuzzer.cc

```diff
@@ -0,0 +1,26 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include "lz4.h"
+
+#define CHECK(COND)   if (!(COND)) { abort(); }
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  size_t const compressed_dest_size = LZ4_compressBound(size);
+  char *const dest_buffer = (char *)malloc(compressed_dest_size);
+
+  if (dest_buffer != NULL)
```

Comment:
We can `CHECK(dest_buffer != NULL)`. It's okay to fail the fuzzer if we can't allocate memory.